lib/launcher.js

```diff
@@ -217,12 +217,13 @@ class Launcher {
             this.logBuffer.add({ level: 'system', msg: 'Updating project dependencies' })
             pkg.dependencies = wantedDependencies
             fs.writeFileSync(pkgFilePath, JSON.stringify(pkg, null, 2))
+            const npmEnv = this.settings.env
             const npmCommand = process.platform === 'win32' ? 'npm.cmd' : 'npm'
             return new Promise((resolve, reject) => {
                 const child = childProcess.spawn(
                     npmCommand,
                     ['install', '--omit=dev', '--no-audit', '--no-update-notifier', '--no-fund'],
-                    { windowsHide: true, cwd: path.join(this.settings.rootDir, this.settings.userDir) })
+                    { windowsHide: true, cwd: path.join(this.settings.rootDir, this.settings.userDir), env: npmEnv })
```

Comment:
The default value of `env` in the spawn call is `process.env` 

I believe be setting this to the content of `settings.env` the process loses and/all other env vars.  

Is that desirable?